scripts/check_storage_keys.sh

```
@@ -12,14 +12,42 @@ echo "Checking for storage key collisions in $CONTRACTS_DIR..."
+        
+        duplicates=$(echo "$const_definitions" | uniq -d)
+        
 
         # Find constant symbol key definitions for collision detection.
         const_definitions=$(grep -r "const .*Symbol = symbol_short!(" "$contract/src" | \
                             sed -n 's/.*symbol_short!("\([^"]*\)").*/\1/p' | \
                             sort)
 
-        duplicates=$(echo "$const_definitions" | uniq -d)
-
+        duplicates=$(echo "$const_definitions" | uniq -
```

Shellcheck (0.11.0)

[error] Couldn't parse this command expansion. Fix to allow more checks.

(SC1073)